sidebars.js

```diff
@@ -76,7 +76,8 @@ const sidebars = {
       label: 'Deploying',
       collapsed: true,
       items: [
-        'deploying/fly_io'
+        'deploying/fly_io',
+        'deploying/vps'
```

Comment:
I think you can change the order here - vps is more important so it should be on top.